website_compassion/template/my_account_write_a_letter.xml

```diff
@@ -126,13 +129,7 @@
                     <span>What happens to my letter?</span>
                 </a>
             </div>
-            <t t-set="placeholder">Write your text here...</t>
-            <t t-if="auto_texts">
-                <textarea id="letter_content" class="w-100 auto_text" style="height: 30rem; resize: none;"><t t-raw="auto_texts[child_id.id]"/></textarea>
-            </t>
-            <t t-else="">
-                <textarea id="letter_content" class="w-100" t-att-placeholder="placeholder" style="height: 30rem; resize: none;"/>
-            </t>
+            <textarea id="letter_content" class="w-100" placeholder="Write your text here..." style="height: 30rem; resize: none;"></textarea>
```

Comment:
Please restore the placeholder as it was done before otherwise the string "Write your text here..." won't be translated on the website.